src/common/classes/TempFile.cpp

```diff
@@ -229,7 +230,7 @@ void TempFile::init(const PathName& directory, const PathName& prefix)
 
 	if (doUnlink)
 	{
-		::unlink(filename.c_str());
+		::do_unlink(filename.c_str());
```

Comment:
Temporary files on Windows are created with "delete on close" flag and never deleted explicitly. So, erasing of files don't, actually, works there.